cmake/projects/Qt/hunter.cmake

```diff
@@ -21,6 +21,17 @@ hunter_add_version(
     PACKAGE_NAME
     Qt
     VERSION
+    "5.7.1"
+    URL
+    "http://download.qt.io/official_releases/qt/5.7/5.7.1/single/qt-everywhere-opensource-src-5.7.1.7z"
```

Comment:
CMake reporting error while trying to unpack this archive:
```
archive_write_data_block()
archive_write_data_block()
archive_write_data_block()
archive_write_data_block()
archive_write_data_block()
archive_write_data_block()
CMake Error: Problem with archive_write_header(): Can't create 'qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/gpu/gles2_conform_support/native/egl_native.cc'
CMake Error: Current file: qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/gpu/gles2_conform_support/native/egl_native.cc
CMake Error: Problem extracting tar: /.../Hunter/_Base/Download/Qt/5.7.1/72ed6f6/qt-everywhere-opensource-src-5.7.1.7z
```

`7z` report some issues too:
```
> 7z e qt-everywhere-opensource-src-5.7.1.7z

7-Zip [64] 9.20  Copyright (c) 1999-2010 Igor Pavlov  2010-11-18
p7zip Version 9.20 (locale=en_US.UTF-8,Utf16=on,HugeFiles=on,4 CPUs)

Processing archive: qt-everywhere-opensource-src-5.7.1.7z

Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/media/tools/layout_tests/result/2011-08-19-11
Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/media/tools/layout_tests/result/2011-08-19-21
Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/media/tools/layout_tests/result/2011-09-11-18
Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/media/tools/layout_tests/result/2011-09-11-19
Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/net/data/url_request_unittest/308-without-location-header
Extracting  qt-everywhere-opensource-src-5.7.1/qttools/tests/auto/linguist/lupdate/testdata/good/proparsing2/a
Extracting  qt-everywhere-opensource-src-5.7.1/qtwebengine/src/3rdparty/chromium/third_party/yasm/source/patched-yasm/ABOUT-NLS
file ABOUT-NLS
already exists. Overwrite with
qt-everywhere-opensource-src-5.7.1/gnuwin32/contrib/libiconv/1.9.2/libiconv-1.9.2-src/ABOUT-NLS?
```